src/templates/RustBufferTemplate.java

```diff
@@ -69,6 +75,19 @@ public void setValue(RustBuffer.ByValue value) {
         pointer.setInt(4, (int) value.len);
         pointer.setPointer(8, value.data);
     }
+
+    /**
+     * Get a `RustBuffer.ByValue` from this reference.
+     */
+    public RustBuffer.ByValue getValue() {
+        Pointer pointer = this.getPointer();
+        RustBuffer.ByValue value = new RustBuffer.ByValue();
+        value.writeField("capacity", pointer.getLong(0));
```

Comment:
And the Rust ones are always 64 bits for capacity and length, regardless of architecture?

Reply:
This is a construct just for usage with uniffi, so yeah the capacity and length is always 64 bit.